pkg/tools/tools.go

```diff
@@ -54,12 +54,20 @@ func ImageImportIntoClusterMulti(ctx context.Context, runtime runtimes.Runtime,
 		found := false
 		// Check if the current element is a file
 		if _, err := os.Stat(image); os.IsNotExist(err) {
+			// `runtimeImages` are returned without the 'docker.io/' prefix. So if `image` has such a prefix, we shall drop it.
+			localImageName := strings.TrimPrefix(image, "docker.io/")
```

Comment:
This will work for now (as k3d currently only supports docker), but will likely break once we support other runtimes as well (e.g. plain containerd would return it with the docker.io prefix), but that's just for future reference :+1: